### DISCUSSION TITLE
Add other pictures than icon in the documtation for better visualization

### DISCUSSION THREAD

**Discussion by flomickl** [apache/streampipes#1316] [Dev]
For a better output visualization, I added a second png

```
## Output

<p align="center">
    <img src="output.png" width="500;"/>
</p>
```

which exists in the resource folder.
![resources_png](https://user-images.githubusercontent.com/16739072/219965947-cf0350a3-89fe-4c53-95cd-56de2396e1a5.png)

Currently, this picture doesn't show up in the documentation. 

![pictures_in_documentation](https://user-images.githubusercontent.com/16739072/219965963-d130c4b2-a3d8-4312-a95f-972950a378fe.png)

Do I miss something? It seems that only the name icon.png can be "rendered "
Do I miss something?

**Comment by bossenti** [comment]
Since the documentation file is a simple Markdown file, it shouldn't be a problem to include another image
I just played around a bit and got it sometimes working with the same code as above and sometimes not, seems to be a bit flaky
@dominikriemer @tenthe do you have any experience here?

**Reply by tenthe** [reply]
Hey,
you sample looks good to me. I have done this before (e.g. [1]).
The only difference I see is the "px" for the width. But I am not sure if this is the reason for the inconsistent behavior.

[1] https://github.com/apache/streampipes/blob/dev/streampipes-extensions/streampipes-processors-transformation-jvm/src/main/resources/org.apache.streampipes.processors.transformation.jvm.booloperator.timekeeping/documentation.md

**Reply by dominikriemer** [reply]
Did you register the icon in the declareModel method?

https://github.com/apache/streampipes/blob/dev/streampipes-extensions/streampipes-processors-transformation-jvm/src/main/java/org/apache/streampipes/processors/transformation/jvm/processor/booloperator/timekeeping/BooleanTimekeepingController.java

**Reply by flomickl** [reply]
ah, good point. I have not done it and probably this will solve the problem.

**Comment by flomickl** [comment]
@dominikriemer @tenthe 
I implemented it as described here
https://github.com/apache/streampipes/blob/6a00daffa45ecd0c63b52a1fcfd09f6798895695/streampipes-extensions/streampipes-processors-geo-jvm/src/main/java/org/apache/streampipes/processors/geo/jvm/jts/processor/derivedgeometry/point/CreateDerivedPointProcessor.java#L69

and in the document here
https://github.com/apache/streampipes/tree/1321-geo-create-derived-point-of-input-geometry-processor/streampipes-extensions/streampipes-processors-geo-jvm/src/main/resources/org.apache.streampipes.processors.geo.jvm.jts.processor.derivedgeometry.point

but still the same as above.
Do I miss anything?

**Reply by tenthe** [reply]
Have you reinstalled the pipeline element?
This is necessary to update the assets.

**Reply by flomickl** [reply]
Yes. I also cleaned my browser cache and because I switched the branches, I also did a StreamPipes clean -v
In the normal markdown preview, I can see the image.

**Comment by nicolasperivolaris** [comment]
Hello,
Does somebody succeeded to put an image in the documentation.md file ?

**Reply by nicolasperivolaris** [reply]
It has been fixed : #3199
